src/shared/styles/formstyles.scss

```diff
@@ -0,0 +1,26 @@
+.form__wrapper {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  .form__input {
+    width: 70%;
+    margin-bottom: 2rem;
```

Comment:
@media only screen and (min-width: 576px) {
  .form__wrapper {
    .form__input {
      width: 403px;
    }
  }
}

you could add media queries here